server.js

```diff
@@ -2897,6 +2897,43 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// ReadTheDocs build
+camp.route(/^\/readthedocs\/([^\/]+)(?:\/(.+))?.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var project = match[1];
+  var version = match[2];
+  var format = match[3];
+  var badgeData = getBadgeData('docs', data);
+  var url = 'https://readthedocs.org/projects/' + project + '/badge/';
+  if (version != null) {
+    url += '?version=' + version;
```

Comment:
For safety, how about URL-encoding this?

Reply:
My bad, good catch. I added `encodeUriComponent()` arround `project` and `version`.